roles/openshift_node/tasks/storage_plugins/main.yml

```diff
@@ -3,11 +3,12 @@
 # additional package dependencies
 - name: NFS storage plugin configuration
   include: nfs.yml
+  when: not is_containerized | bool
 
 - name: GlusterFS storage plugin configuration
   include: glusterfs.yml
-  when: "'glusterfs' in openshift.node.storage_plugin_deps"
+  when: ( "'glusterfs' in openshift.node.storage_plugin_deps" ) and not is_containerized | bool
```

Comment:
parens are not needed here, just need to include the second conditional in the quotes:
```
"'glusterfs' in openshift.node.storage_plugin_deps and not is_containerized | bool"
```